### ISSUE TITLE
Cannot GET /index.bundle

### ISSUE THREAD

**Issue by jurajkrivda** [callstack/haul#245]
Hello. I tried to use haul with fresh install of RN 0.49.1 but I got this error

**Comment by Krizzu** [comment]
Hey @jurajkrivda, thanks for creating this issue.

RN 0.49 has introduced a breaking change in file naming - [5d4c6e5](https://github.com/facebook/react-native/commit/5d4c6e5f23e3f7f64576cccff76822d4b7635ab1)

Will looking into it soon

**Comment by vlad-zhukov** [comment]
A simple workaround for 0.49:

```js
module.exports = (_, {output, module}) => ({
    entry: './index.js',
    output: {
        ...output,
        filename: 'index.bundle',
    },
});

```

**Comment by vlad-zhukov** [comment]
There is also another error:
`Expected Content-type to be 'application/javascript' or 'text/javascript', but got 'application/octec-stream; charset=UTF-8'.`

This can be fixed by adding `headers: {'Content-Type': 'application/javascript'}` to `webpackDevMiddleware` config in `src/server/index.js`.

**Comment by Krizzu** [comment]
Thanks, fixed in #246
